index.html

```diff
@@ -642,12 +646,25 @@
       </p>
       <pre class="idl">
         interface PushManager {
+          [SameObject] static readonly attribute FrozenArray&lt;DOMString&gt; supportedContentEncodings;
+
           Promise&lt;PushSubscription&gt; subscribe(optional PushSubscriptionOptionsInit options);
           Promise&lt;PushSubscription?&gt; getSubscription();
           Promise&lt;PushPermissionState&gt; permissionState(optional PushSubscriptionOptionsInit options);
         };
       </pre>
       <p>
+        The <dfn>supportedContentEncodings</dfn> attribute exposes the sequence of supported content
+        codings using which the payload of a <a>push message</a> can be encrypted. When a content
+        coding is used, it MUST be set in the <a>Content-Encoding</a> HTTP header when sending a
+        <a>push message</a> to the <a>push service</a>.
```

Comment:
I don't think that you want the MUST requirement here.

"A content coding is indicated using the <a>Content-Encoding</a> header field when requesting the sending of a <a>push message</a> from the <a>push service</a>."   ...Maybe.